src/Validation/ConstraintViolationException.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * This file is part of Lcobucci\JWT, a simple library to handle JWT and JWS
+ *
+ * @license http://opensource.org/licenses/BSD-3-Clause BSD-3-Clause
+ */
+
+declare(strict_types=1);
+
+namespace Lcobucci\JWT\Validation;
+
+use Lcobucci\JWT\Exception;
+
+/**
+ * @author Luís Otávio Cobucci Oblonczyk <lcobucci@gmail.com>
+ * @author Danny Dörfel <danny.dorfel@gmail.com>
+ * @author Marco Piveta <ocramius@gmail.com>
```

Comment:
Sorry by the typo.

You helped on #75 discussion... IMO you're a co-author 😄 (but I can remove it if you want)